deerhunt/src/components/ComingSoonCard.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import {
+  Box,
+  Center,
+  Heading,
+  Text,
+  Stack,
+  Avatar,
+  useColorModeValue,
+  Image,
+} from "@chakra-ui/react";
+import BattleCodeLogo from "../images/ComingSoonLogo.png";
+import MCSSLogo from "../images/MCSSLogo.png";
+import UTMRoboticsLogo from "../images/UTMRoboticsLogo.png";
+import CountDownTimer from "./Countdown";
+
+export default function ComingSoonCard() {
```

Comment:
This is good! We'll adapt this later to use the api to get the information you have here.